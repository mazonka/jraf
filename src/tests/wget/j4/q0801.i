command=jw md 0 /.jraf.sys/users : login admin@server.com * : login admim@server.com * : read f82d3bb17612458c /.jraf.sys/users : read f82d3bb17612458c /.jraf.sys/users/admin@server.com : read f82d3bb17612458c /.jraf.sys/users/admin@server.com/counter : read f82d3bb17612458c /.jraf.sys/users/admin@server.com/quota : read f82d3bb17612458c /.jraf.sys/users/admin@server.com/uname : read 5aeb1f6c5cd9c44a /.jraf.sys/users/admim@server.com/uname