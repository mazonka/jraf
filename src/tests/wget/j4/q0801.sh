#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
echo ""
echo "=="
pushd $wr >/dev/null
ls -1 .jraf.sys/users/admin@server.com/access 
popd >/dev/null
rm -rf $wr
