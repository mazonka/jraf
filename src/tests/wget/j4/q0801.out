OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf? : OK http://127.0.0.1:16000/jraf? : OK 0 -1 1 0 -1 admin@server.com : OK 0 -1 4 0 15 access 0 2 counter 0 6 quota 0 17 uname : OK 0 2 Mwo= : OK 0 6 MTAwMDAK : OK 0 17 MzRkM2FlN2M1ZTJjZGViYgo= : JRAF_FAIL system path
==
.jraf.sys/users/admin@server.com/access
